app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def log_out
+    session.delete(:user_id)
+    @current_user = nil
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
Just a bit better:
```ruby
@current_user ||= User.find(session[:user_id])
```

Reply:
Read about it and find raises and error while find_by would return nil. Is it better to handle the exception?

Reply:
Fixed